Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -1139,6 +1139,23 @@ void AODProducerWorkflowDPL::fillMCTrackLabelsTable(MCTrackLabelCursorType& mcTr
                 }
               }
             }
+            if (trackIndex.includesDet(DetID::ITS)) {
+              auto itsGID = data.getITSContributorGID(trackIndex);
+              if (itsGID.isIndexSet()) {
```

Comment:
This check is not needed, if `trackIndex.includesDet(DetID::ITS)` is true, then the `itsGID.isIndexSet()` is guaranteed.
Just check
```
if (itsGID.getSource() == GIndex::ITS) { 
 ... as ITS track
} else {
 ... as AB track
}
```